### PR TITLE
[Rules] Add Configuration Rules

### PR DESCRIPTION
**Pull request by pedrosousa**
Addresses PCX-1996.

### CODE REVIEW

**Comment by pedrosousa** [comment]
Preview of new section:
https://pedro-pcx-1996-rules-configu.cloudflare-docs-7ou.pages.dev/rules/configuration-rules/

File: content/rules/configuration-rules/create-api.md [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+    * **kind**: `zone`
+    * **phase**: `http_config_settings`
+
+3. Use the [Update ruleset](/ruleset-engine/rulesets-api/update/) method to add a Configuration Rule to the list of ruleset rules (check the examples below). Alternatively, include the rule in the [Create ruleset](/ruleset-engine/rulesets-api/create/) request mentioned in the previous step.
+
+## Required API token permissions
+
+The API token used in API requests to manage Configuration Rules must have at least the following permission:
+
+* _Zone_ > _Config Settings_ > _Edit_
```
Comment (marciocloudflare) [inline content/rules/configuration-rules/create-api.md]: Are these drop-down menus? Is that why they're italic?

File: content/rules/configuration-rules/_index.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
+---
+pcx_content_type: concept
+title: Configuration Rules
```
Comment (marciocloudflare) [inline content/rules/configuration-rules/_index.md:3]: Add `(beta)` to title

File: content/rules/configuration-rules/create-dashboard.md [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+pcx_content_type: how-to
+title: Create a rule in the dashboard
+weight: 3
+---
+
+# Create a Configuration Rule in the dashboard
+
+1. Log in to the [Cloudflare dashboard](https://dash.cloudflare.com) and select your account and domain.
+2. Go to **Rules** > **Configuration Rules**.
+3. Click **Create configuration rule**.
```
Comment (marciocloudflare) [inline content/rules/configuration-rules/create-dashboard.md]:
```suggestion
3. Select **Create configuration rule**.
```

File: content/rules/configuration-rules/create-dashboard.md [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+4. Enter a descriptive name for the rule in **Rule name**.
+5.  Under **When incoming requests match**, define the [rule expression](/firewall/cf-dashboard/edit-expressions/).
```
Comment (marciocloudflare) [inline content/rules/configuration-rules/create-dashboard.md]:
```suggestion
5. Under **When incoming requests match**, define the [rule expression](/firewall/cf-dashboard/edit-expressions/).
```

File: content/rules/configuration-rules/create-dashboard.md [hunk 5]
```diff
@@ -0,0 +1,15 @@
...
+6. Under **Then the settings are**, define the [configuration settings](/rules/configuration-rules/settings/) you wish to change for requests matching the rule expression.
+7.  To save and deploy your rule, click **Deploy**. If you are not ready to deploy your rule, click **Save as Draft**.
```
Comment (marciocloudflare) [inline content/rules/configuration-rules/create-dashboard.md]:
```suggestion
7. To save and deploy your rule, select **Deploy**. If you are not ready to deploy your rule, select **Save as Draft**.
```

File: content/ruleset-engine/rules-language/actions.md [hunk 6]
```diff
@@ -202,6 +202,26 @@ The available actions depend on the [phase](/ruleset-engine/about/phases/) where
...
+      </td>
+      <td>
+        <p>
+          Changes the configuration settings of one or more Cloudflare products.
+        </p>
+        <p>
+          Only available for <a href="/rules/configuration-rules/">Configuration Rules</a>, in the <code>http_config_settings</code> phase.
+        </p>
+        <p>
+          In the Cloudflare dashboard, this action is not listed in action selection dropdowns. To use this action, create a Configuration Rule.
```
Comment (marciocloudflare) [inline content/ruleset-engine/rules-language/actions.md]: Should `...create a Configuration Rule` link to the appropriate page?

**Review by marciocloudflare — APPROVED** [review]

File: content/rules/configuration-rules/create-api.md (hunk 1 above)
Comment (pedrosousa) [reply content/rules/configuration-rules/create-api.md]: Yes, they appear in the Create custom API token page in the dashboard. Unfortunately the API names are slightly different, so I'm using the dashboard values -- which have changed (I'll update the PR)!
image

File: content/ruleset-engine/rules-language/actions.md (hunk 6 above)
Comment (pedrosousa) [reply content/ruleset-engine/rules-language/actions.md]: There's a link 3 lines before this one, but it's for the section's main page — I'll add a link.

**Pull request merged**